input/gcamdata/inst/extdata/gcam-india/PRIMA_pop_India.csv

```diff
@@ -0,0 +1,78 @@
+# File: PRIMA_pop_India.csv,,,,,,,,,,,,,,,,,,,,,
+# Title: 2005-2100 population projections by state,,,,,,,,,,,,,,,,,,,,,
+# Units: Persons,,,,,,,,,,,,,,,,,,,,,
+# Source: Interpolation from the IIASA estimates,,,,,,,,,,,,,,,,,,,,,
+state,state_name,2005,2010,2015,2020,2025,2030,2035,2040,2045,2050,2055,2060,2065,2070,2075,2080,2085,2090,2095,2100
```

Comment:
need column type here

Reply:
Sure. We'll remove the commas from the terminals.

Reply:
@rynacui, I had a query regarding the row "#------" which is present in all the files. What exactly it corresponds to and what is its role? and Are the no. of '-' are corresponding to the no. of columns?